lib/hybrid-sdk/common/utils/token.ts

```diff
@@ -23,6 +23,6 @@ export function maskSCMToken(token) {
 }
 
 export const extractBrokerTokenFromUrl = (urlString) => {
-  const regex = /^\/broker\/([a-z0-9-]+)\//;
+  const regex = /\/broker\/([a-z0-9-]+)\//;
```

Comment:
I think we can easily unit-test this one.

Reply:
fixed